proxygen/httpserver/samples/echo/EchoServer.cpp

```diff
@@ -44,18 +50,45 @@ class EchoHandlerFactory : public RequestHandlerFactory {
   }
 
   RequestHandler* onRequest(RequestHandler*, HTTPMessage*) noexcept override {
+    numThreads++;
+    iotpe_->setNumThreads(numThreads.load());
```

Comment:
This seems quite strange -- to increase the number of available threads on every request, with no bound?